rlog/2024-07-01-bloomfilter.mdx

```diff
@@ -132,11 +131,99 @@ This means that querying the sliding-window Bloom filter will yield different re
 
 This can be done with $k$ hash functions as we used earlier.
 Alternatively, Szepieniec and Værge proposed using the same hash function but to produce $k$ entries in the current window.
-Specifically, we obtain the bits we wish to flip to 1's for entry $x$ by computing $h(x || i)$ for each $i \in \{0,\dots, k-1\}$.
+Specifically, we obtain the bits we wish to flip to 1's by computing $h(X || i)$ for each $i \in \{0,\dots, k-1\}$ and $X$ as we will define next.
+For Sliding-Window Bloom filters, $X$ is more than just the element we wish to append to the filter.
+Instead, $X$ consists of the message $x$ and a timestamp $t$. 
```

Comment:
I think this is the first mention of the word `message`, can you expand?

Reply:
Should be element. Thanks!